cli/cli/kurtosis_config/resolved_config/kurtosis_cluster_config_test.go

```diff
@@ -144,22 +160,104 @@ func TestNewKurtosisClusterConfigLogsAggregatorFullConfig(t *testing.T) {
+	kubernetesFullConfig := v5.KubernetesClusterConfigV5{
+		KubernetesClusterName:  &kubernetesClusterName,
+		StorageClass:           &kubernetesStorageClass,
+		EnclaveSizeInMegabytes: &kubernetesEnclaveSizeInMB,
+		EngineNodeName:         &kubernetesEngineNodeName,
+	}
+	kurtosisClusterConfigOverrides := v5.KurtosisClusterConfigV5{
+		Type:   &kubernetesType,
+		Config: &kubernetesFullConfig,
+		LogsAggregator: &v5.LogsAggregatorConfigV5{
+			Sinks: map[string]map[string]interface{}{
+				"elasticsearch": {
+					"type": "elasticsearch",
+				},
+			},
+		},
+		GraflokiConfig:               nil,
+		ShouldTurnOffDefaultLogsSink: nil,
+	}
+	_, err := NewKurtosisClusterConfigFromOverrides("test", &kurtosisClusterConfigOverrides)
+	require.NoError(t, err)
+}
+
+func TestNewKurtosisClusterConfigGraflokiNoConfig(t *testing.T) {
+	kubernetesType := KurtosisClusterType_Kubernetes.String()
+	kubernetesClusterName := "some-name"
+	kubernetesStorageClass := "some-storage-class"
+	kubernetesEnclaveSizeInMB := uint(5)
+	kubernetesEngineNodeName := "some-node-name"
+	kubernetesFullConfig := v5.KubernetesClusterConfigV5{
 		KubernetesClusterName:  &kubernetesClusterName,
 		StorageClass:           &kubernetesStorageClass,
 		EnclaveSizeInMegabytes: &kubernetesEnclaveSizeInMB,
 		EngineNodeName:         &kubernetesEngineNodeName,
 	}
-	kurtosisClusterConfigOverrides := v4.KurtosisClusterConfigV4{
+	kurtosisClusterConfigOverrides := v5.KurtosisClusterConfigV5{
 		Type:   &kubernetesType,
 		Config: &kubernetesFullConfig,
-		LogsAggregator: &v4.LogsAggregatorConfigV4{
+		LogsAggregator: &v5.LogsAggregatorConfigV5{
 			Sinks: map[string]map[string]interface{}{
 				"elasticsearch": {
 					"type": "elasticsearch",
 				},
 			},
 		},
+		GraflokiConfig:               nil,
+		ShouldTurnOffDefaultLogsSink: nil,
+	}
+	_, err := NewKurtosisClusterConfigFromOverrides("test", &kurtosisClusterConfigOverrides)
+	require.NoError(t, err)
+}
+
+func TestNewKurtosisClusterConfigGraflokiFullConfig(t *testing.T) {
+	kubernetesType := KurtosisClusterType_Kubernetes.String()
+	kubernetesClusterName := "some-name"
+	kubernetesStorageClass := "some-storage-class"
+	kubernetesEnclaveSizeInMB := uint(5)
+	kubernetesEngineNodeName := "some-node-name"
+	kubernetesFullConfig := v5.KubernetesClusterConfigV5{
+		KubernetesClusterName:  &kubernetesClusterName,
+		StorageClass:           &kubernetesStorageClass,
+		EnclaveSizeInMegabytes: &kubernetesEnclaveSizeInMB,
+		EngineNodeName:         &kubernetesEngineNodeName,
+	}
+	kurtosisClusterConfigOverrides := v5.KurtosisClusterConfigV5{
+		Type:           &kubernetesType,
+		Config:         &kubernetesFullConfig,
+		LogsAggregator: nil,
+		GraflokiConfig: &v5.GraflokiConfig{
+			ShouldStartBeforeEngine: false,
+			GrafanaImage:            "grafana:1.32",
+			LokiImage:               "loki:1.32",
+		},
+		ShouldTurnOffDefaultLogsSink: nil,
+	}
+	_, err := NewKurtosisClusterConfigFromOverrides("test", &kurtosisClusterConfigOverrides)
```

Comment:
Prob want to have asserts that your overrides are getting picked up (to guard against the bug earlier in this PR, with overrides not being used on the Kubernetes version)